HometaskPurchaseInStore/HWPurchaseInStore/Services/PlaceOrder.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HWPurchaseInStore.Services
+{
+    internal class PlaceOrder
+    {
+        public string GenerateID()
```

Comment:
we can write a short form for this method 'public string ID => Guid.NewGuid().ToString();'